app/controllers/application_controller.rb

```diff
@@ -1,7 +1,21 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user = User.first
-  end
+  # def current_user
+  #   @current_user = User.first
+  # end
+
+  # helper_method :current_user
```

Comment:
- [ ] [OPTIONAL] It is not good practice to commit old pieces of code as comments, please remove them from the codebase :pray: